samples/smp_svr/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.20.0)
+
+# Standard Zephyr application boilerplate.
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(smp_svr)
+
+target_sources(app PRIVATE ../../../zephyr/samples/subsys/mgmt/mcumgr/smp_svr/src/main.c)
+target_sources(app PRIVATE ../../../zephyr/samples/subsys/mgmt/mcumgr/smp_svr/src/bluetooth.c)
```

```suggestion
set(smp_srv_dir $ENV{ZEPHYR_BASE}/samples/subsys/mgmt/mcumgr/smp_svr)
target_sources(app PRIVATE ${smp_srv_dir}/src/main.c)
target_sources_ifdef(CONFIG_MCUMGR_TRANSPORT_BT app PRIVATE ${smp_srv_dir}/src/bluetooth.c)
```
you can use ZEPHYR_BASE to point the original code and use variable to not repeat it.
Also include bluetooth.c conditionally, as it is done in the original config